packages/access-api/test/store-list.js

```diff
@@ -69,85 +69,4 @@ describe('proxy store/list invocations to upload-api', function () {
       assert.ok(!result?.error, 'should not be an error')
     })
   }
-
-  it('errors when a bad delegation is given as proof', async () => {
```

Comment:
When I wrote this test, I think it only passed because of the ucanto audience check.
This test no longer makes sense because the final way we did the upload-api proxying doesn't do any signature verification here in access-api, it just proxies to w3infra upload-api and does the validation there.
But this test stubbed that out in a way that didn't even do that, so there was no reason for it to pass.